docs/reference/cat/tasks.asciidoc

```diff
@@ -13,9 +13,9 @@ similar to the <<tasks,task management>> API.
 [[cat-tasks-api-request]]
 ==== {api-request-title}
 
-`GET /_cat/_tasks/<task_id>`
+`GET /_cat/tasks/<task_id>`
```

Comment:
I just tried this on 7.4.2 and get something along the lines of 
```
#! Deprecation: [types removal] Specifying types in document get requests is deprecated, use the /{index}/_doc/{id} endpoint instead.
{
  "error" : {
    "root_cause" : [
      {
        "type" : "index_not_found_exception",
        "reason" : "no such index [_cat]",
        "resource.type" : "index_expression",
        "resource.id" : "_cat",
        "index_uuid" : "_na_",
        "index" : "_cat"
      }
    ],
    "type" : "index_not_found_exception",
    "reason" : "no such index [_cat]",
    "resource.type" : "index_expression",
    "resource.id" : "_cat",
    "index_uuid" : "_na_",
    "index" : "_cat"
  },
  "status" : 404
}
```
which to me looks like the request is hitting the wrong endpoint (trying to index a document). 
If this is the case, this might either be a bug we should file, or if getting tasks by id via the _cat API isn't supposed to be working we should remove this from the docs. Maybe @jrodewig or @imotov have an idea
how to proceed.

Reply:
@cbuescher I'm seeing the same error on my end. I'll remove individual task retrieval from the docs. Thanks for raising this!